weather_mv/loader_pipeline/ee.py

```diff
@@ -294,8 +303,26 @@ def validate_arguments(cls, known_args: argparse.Namespace, pipeline_args: t.Lis
                             region=pipeline_options_dict.get('region'))
             logger.info('Region validation completed successfully.')
 
+        # Check for the band_names json file
+        if not os.path.exists(known_args.band_names):
```

Comment:
**--band_names** is not a required argument. So we'd want to check for this only when the user has provided one.

The condition should be like this:
`if known_args.band_names and not os.path.exists(known_args.band_names):`